### ISSUE TITLE
Create list of subnets for the secops team

### ISSUE THREAD

**Issue by tbaker1313** [BCDevOps/developer-experience#3696]
**Describe the issue**
Create a list of subnets for Secops describing the subnets for each of our clusters.

**How does this benefit the users of our platform?**
Helpful when Secops needs to determine what application/pod is causing a security issue.

**Definition of done**

- [ ] Provide a list of subnets to Secops

**Comment by tmorik** [comment]
Ref: 
- old spreadsheet ( Needs to be updated): https://docs.google.com/spreadsheets/d/17TEPNx5_UbH-ZbPMChcw9Px6vbyR1-gIIuAc9D7ae-I/edit?pli=1#gid=0

**Comment by tmorik** [comment]
Asking network team for vlan list.

**Comment by tmorik** [comment]
Yes. that's what they recommended also. (I remembered they used to have a spreadsheet like this, but obviously, there is no more such thing.) I will do this step by step as suggested.

**Comment by tmorik** [comment]
Spreadsheet (drafting): https://advsolcan.sharepoint.com/:x:/r/sites/ManagedContainerServicesMCS/Shared%20Documents/Daily%20Ops/BCGov-OCP4%20Networks.xlsx?d=w1586de164cd64200997b0567d2858730&csf=1&web=1&e=6uE5iw

Check steps:
1. Get node IPs from clusters (using `oc get hostsubnets` command)
2. Get subnet info from infoblox and import it to the spreadsheet
3. Get network info from infoblox and import it to the spreadsheet
4. Look up `PROD`, `MGMT` and `STOR` subnet addresses, vlan ids from the imported infoblox info.
5. Add extra info such as `app`, `api` ip addresses for each clusters.

Each clusters columns will be...
- PROD HOST NAME from node
- PROD IP from node
- PROD IP SUBNET in infoblox
- PROD SUBNET VLAN ID in infoblox
- MGMT HOSTNAME - dns lookup
- MGMT IP from node
- MGMT IP SUBNET in infoblox
- MGMT IP SUBNET VLAN ID in infoblox
- STORAGE IP from node
- STORAGE IP SUBNET in infoblox
- STORAGE IP SUBNET VLAN ID in infoblox
- HOSTSUBNET -  the container subnet network on a node
- ROLE
- Remarks

**Comment by tmorik** [comment]
Spreadsheet updated. Continue on NSX cluster info.

**Comment by tmorik** [comment]
Added NSX cluster info.  For NSX clusters, PROD IP SUBNET and VLAN ID columns changed as below:

- PROD IP SUBNET in infoblox  --> PROD IP SUBNET in NSX Console
- PROD SUBNET VLAN ID in infoblox --> PROD SUBNET SEGMENT NAME in NSX Console

**Comment by tmorik** [comment]
Cleanup spreadsheet and removed formulas from the spreadsheet for SecOps Team.

https://advsolcan.sharepoint.com/:x:/r/sites/ManagedContainerServicesMCS/Shared%20Documents/Daily%20Ops/BCGov-OCP4%20Networks_for_SecOps.xlsx?d=w06c3d70fc57f4e65b65525703bccfd00&csf=1&web=1&e=ICZ55T

I will pass this to Nick and Avneet for their review.

**Comment by tmorik** [comment]
The spreadsheet has been sent.
